src/nupic/bindings/algorithms.i

```diff
@@ -1507,6 +1509,170 @@ inline PyObject* generate2DGaussianSample(nupic::UInt32 nrows, nupic::UInt32 nco
 
 }
 
+
+%include <nupic/algorithms/SDRClassifier.hpp>
+
+%pythoncode %{
+  import numpy
+%}
+
+%extend nupic::algorithms::sdr_classifier::SDRClassifier
+{
+  %pythoncode %{
+    VERSION = 1
+
+    def __init__(self, steps=(1,), alpha=0.001, actValueAlpha=0.3, verbosity=0):
+      self.this = _ALGORITHMS.new_SDRClassifier(
+          steps, alpha, actValueAlpha, verbosity)
+      self.valueToCategory = {}
+      self.version = SDRClassifier.VERSION
+
+    def compute(self, recordNum, patternNZ, classification, learn, infer):
```

Comment:
@scottpurdy: why aren't we including docstrings in these python interface methods with all the normally-required stuff, such as parameter and return value documentation?